Documentation/cmdref/cilium-agent.md

```diff
@@ -319,6 +319,7 @@ cilium-agent [flags]
       --vtep-endpoint strings                                     List of VTEP IP addresses
       --vtep-mac strings                                          List of VTEP MAC addresses for forwarding traffic outside the cluster
       --vtep-mask string                                          VTEP CIDR Mask for all VTEP CIDRs (default "255.255.255.0")
+      --wireguard-persistent-keepalive duration                   Controls Wireguard PersistentKeepalive option
```

Comment:
This should indicate the unit for this option. Is it seconds? If so, suggested:
```
The Wireguard keepalive interval in seconds
```

Reply:
Ah, I see it can be interpreted as a duration string. In that case:
```
The Wireguard keepalive interval as a Go duration string
```

Reply:
/cc @asauber Updated. PTAL. Thanks~